planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/src/utilities.cpp

```diff
@@ -439,6 +444,35 @@ double getDistanceBetweenPredictedPaths(
   return min_distance;
 }
 
+double getDistanceBetweenPredictedPathAndObject(
+  const autoware_perception_msgs::msg::DynamicObject & object, const PredictedPath & ego_path,
+  const double start_time, const double end_time, const double resolution,
+  const rclcpp::Logger & logger, const rclcpp::Clock::SharedPtr & clock)
+{
+  rclcpp::Duration t_delta(resolution);
+  double min_distance = std::numeric_limits<double>::max();
+  rclcpp::Time ros_start_time = clock->now() + rclcpp::Duration(start_time);
```

Comment:
I guess it's better to use from_seconds function.

Reply:
Fixed.